client/src/components/mainContent.jsx

```diff
@@ -68,6 +82,7 @@ const MainContent = () => {
               align="right"
               className={classes.button}
               href="#"
+              onClick={() => OpenCreateMeetingDialog()}
```

Comment:
```suggestion
              onClick={OpenCreateMeetingDialog}
```

since you're not passing any events into `OpenCreateMeetingDialog`, you can just pass in the function and it will execute itself on click